include/zephyr/net/mld.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/** @file
+ * @brief Multicast Listener Discovery API
+ */
+
+#ifndef ZEPHYR_INCLUDE_NET_MLD_H_
+#define ZEPHYR_INCLUDE_NET_MLD_H_
+
+/**
+ * @brief MLD (Multicast Listener Discovery)
+ * @defgroup mld Multicast Listener Discovery API
+ * @since 1.8
```

Comment:
I understand the API existed before as internal but it seems odd to state 1.8 here since strictly speaking it's being introduced only now as public API?

Reply:
The API could have been called earlier too by the application even if it was internal, so I think it is ok to have the version set when the API was created.